iroha/src/torii.rs

```diff
@@ -209,11 +209,13 @@ async fn handle_request(state: State<ToriiState>, request: Request) -> Result<Re
                 Ok(Response::InternalError)
             }
         },
-        uri::QUERY_URI => match QueryRequest::try_from(request.payload().to_vec()) {
-            Ok(request) => match request
-                .query
-                .execute(&*state.read().await.world_state_view.read().await)
-            {
+        uri::QUERY_URI => match SignedQueryRequest::try_from(request.payload().to_vec()) {
+            //TODO: check query permissions based on signature?
```

Comment:
Yes, please check that we have issue for that.

Reply:
There was no issue for this so I created it: https://jira.hyperledger.org/browse/IR-872